scip_indexer/SCIPGemMetadata.cc

```diff
@@ -169,4 +178,85 @@ pair<GemMetadata, vector<GemMetadataError>> GemMetadata::readFromConfig(const Fi
     return {GemMetadata(name.value_or(currentDirName()), version.value_or("latest")), errors};
 }
 
+GemMapping::GemMapping() {
+    // The 'ruby' namespace is reserved by RubyGems.org, so we can use that here.
+    // Unfortunately, there are already gems which use the names stdlib and ruby-stdlib,
+    // so not using those here.
+    // FIXME: How do we get the version reliably?
+    this->stdlibGem = make_shared<GemMetadata>(GemMetadata::tryParse("ruby@2.7.2").value());
```

Comment:
Need to set this properly.

Reply:
Also, what should we do about this showing up in snapshot output too? Redact it as well, and maybe have a magic comment to not redact it so that we can test it?